### ISSUE TITLE
Repair lighting to have less shadow / directionality

### ISSUE THREAD

**Issue by Barasakar** [gelic-idealab/impress#4]
**Enhancement Description**
Change the light setting for the scene so that it has less shadow and less directional light. The scene's lighting should be fairly similar to the space's. However, the lighting should not overly affect the colors of models.

**How to Test**
Action: compare a model's color inside and outside of the scene and see how much color variations it has. 
Expected Result: the model should be darker (but the color should be still there) and the lighting will be more ambient.

**Comment by jplaney** [comment]
As a testing platform with was achieved my modifying the lighting so that only the indirect light was used. The idea here is that they will be predicting where shadow may fall and sketching it themselves, so we didn't want them to be confused by shadows cast from an environment light source they can't control.

**Comment by Barasakar** [comment]
I made some changes to the lighting. Let me know what you think @jplaney @parseccentric 

Before: https://i.imgur.com/CZgvqXq.png

After: https://i.imgur.com/cg2tTF3.png

**Comment by jplaney** [comment]
Yeah this is better. It might just be the view angle here, but is there a way to sort of soften/blend that band that still appears a bit between the lit and unlit faces?

**Comment by Barasakar** [comment]
@jplaney Yes, that can be fixed. 

Also I have a question about your opinions on the color of models. For now the models appear a bit darker than the actual material does. Here is how different they are:
Color of a model appears in Komodo:  https://i.imgur.com/8en5xwa.png
Color of the model appears outside of Komodo: https://i.imgur.com/aumlgXO.png

What's your thought on this? Would you want me to make some changes for this?

**Comment by jplaney** [comment]
That second image 404's for me. But the first is a bit dark, I might be fighting two things here where I want indirect lighting but also the shapes should still pop out relative to the starfield.

Here's an example from the early tiltbrush stuff I was doing in regard to lighting (these are on pretty low poly models so might not be the best comparison):

Too direct: https://imgur.com/CLGYwLe
Toned down: https://imgur.com/0TI8ONh

**Comment by Barasakar** [comment]
Sorry about the second image; this one should work https://i.imgur.com/0De2Hfu.png

Based on the images you sent, I think the lighting in the current change should be good to go. I apologized for picking a dark brown color; that was why it appeared darker.

Here is a better example of what the current lighting is like: https://i.imgur.com/kB8UskW.png   
The palette shows what the color actually looks like without lighting, and the sphere shows the color in Komodo with lighting effects.

**Comment by jplaney** [comment]
Yeah, I agree...we can test it out to be sure, but this looks good to me, thanks!
